bom_stock/README.rst

```diff
@@ -0,0 +1,53 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+=================================================================
+Compute Product Stock from its Bill of Material
+=================================================================
+
+Compute the BOM stock Value based on:
+(`Reference stock` of `Product` + How much could I produce of that 
+`Product` according to the component's `Reference Stock`)
+
+This reference stock can be chosen by company through a selection field
+and can be one of the available stock quantity computed in the system :
+Available stock, Virtual stock, immediately_usable stock 
+(from stock_available_immediately).
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues
+<https://github.com/OCA/product-kitting/issues>`_. In case of trouble, please
+check there if your issue has already been reported. If you spotted it first,
+help us smashing it by providing a detailed and welcomed `feedback
+<https://github.com/OCA/
+product-kitting/issues/new?body=module:%20
+bom_stock%0Aversion:%20
+8.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Images
+------
+
+* Odoo Community Association: `Icon <https://github.com/OCA/maintainer-tools/blob/master/template/module/static/description/icon.svg>`_.
+
+Contributors
+------------
+Camptocamp SA
+Sodexis <dev@sodexis.com>
```

Comment:
@pedrobaeza 
do you mean the contributors available in this github repo?

Users who have contributed to this file
    @gurneyalex gurneyalex
    @yvaucher yvaucher
    @guewen guewen
    @sbidoul sbidoul